deliverables/ruby_monk_problems/palindrome.rb

```diff
@@ -1,3 +1,3 @@
 def palindrome?(sentence)
-  # Write your code here
+  sentence.downcase.gsub(" ", "")==sentence.downcase.gsub(" ",    "").reverse
```

Comment:
be careful with your spacing, there are too many spaces in the second gsub.

ideally, you wouldn't have to downcase and gsub sentence again. one way to do this and keep it on one line would be:

` (sentence = sentence.downcase.gsub(" ", ""))==sentence.reverse`